bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/GnirsPrinter.java

```diff
@@ -126,11 +110,6 @@ private void writeSpectroscopyOutput(final UUID id, final GnirsSpectroscopyResul
     }
 
     private String gnirsToString(final Gnirs instrument, final Parameters p) {
-        //Used to format the strings
-        final FormatStringWriter device = new FormatStringWriter();
-        device.setPrecision(3);  // Two decimal places
-        device.clear();
-
 
         String s = "Instrument configuration: \n";
```

Comment:
Is there a reason you build the string like this instead of using a `StringBuilder` as you do in other places? Doesn't seem efficient.